src/config_loader.sh

```
@@ -0,0 +1,266 @@
+    yaml_to_json "$default_config"
+}
+
+# ユーザー設定を読み込み
+load_user_config() {
+    local user_config="${USER_CONFIG_DIR}/config.yml"
+    yaml_to_json "$user_config"
+}
+
+# 設定をマージして読み込み
+load_config() {
+    local default_config=$(load_default_config)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)